src/facade/dragonfly_connection.cc

```diff
@@ -602,6 +602,7 @@ auto Connection::ParseMemcache() -> ParserStatus {
   MCReplyBuilder* builder = static_cast<MCReplyBuilder*>(cc_->reply_builder());
 
   do {
+    builder->SetNoreply(false);
```

Comment:
I think you can move this logic solely into `DispatchMC` and dragonfly_connection should not change.

Reply:
Thanks.